routers/repo/issue.go

```diff
@@ -575,6 +611,19 @@ func NewIssue(ctx *context.Context) {
 	ctx.HTML(200, tplIssueNew)
 }
 
+// NewIssueChoose render creating issue from template page
+func NewIssueChoose(ctx *context.Context) {
```

Comment:
Not liking this name. Choose what exactly?

NewIssueWithTemplate? NewIssueWithTemplateChoice? Issue template choose?

Reply:
`NewIssueWithTemplate` or `NewIssueFromTemplate`?